index.js

```diff
@@ -97,9 +97,19 @@ function filter (obj, { protoAction = 'error', constructorAction = 'error', safe
   return obj
 }
 
+function parse (text, reviver, options) {
+  Error.stackTraceLimit = 0
+  const result = _parse(text, reviver, options)
+  Error.stackTraceLimit = 10
```

Comment:
What if the user does not have the default value?

Reply:
I'd store the `Error.stackTraceLimit` into a variable and then reset to it instead of hardcoding 10.

Reply:
I realize that i dont reset the stackTraceLimit, when i throw an error.

Reply:
done